### PR TITLE
Visual studio snippets

### PR DESCRIPTION
**Pull request by switch87**
snippets from visual studio vc# snippets

### CODE REVIEW

File: snippets/cs.snippets [hunk 1]
```diff
@@ -90,10 +94,10 @@ snippet ?
 snippet do
 	do {
 		${0}
-	} while (${1});
+	} while (${1:True});
```
Comment (lpil) [inline snippets/cs.snippets]: How come this True is uppercase...

File: snippets/cs.snippets [hunk 2]
```diff
@@ -90,10 +94,10 @@ snippet ?
...
 # while loop
 snippet wh
-	while (${1}) {
+	while (${1:true}) {
```
Comment (lpil) [inline snippets/cs.snippets]: ...while this true is lowercase?

**Comment by switch87** [comment]
because I made a mistake ;) I fixed it now. Sorry!

File: snippets/cs.snippets [hunk 3]
```diff
@@ -372,3 +385,89 @@ snippet <rem
...
+# try
+snippet try
+    try{
+        ${0}
+    }
+    catch (${1:System.Exception}){
+        throw;
+    }
+snippet tryf
+    try{
```
Comment (lpil) [inline snippets/cs.snippets]: Shouldn't there be a space before the `{` to match the other snippets?

Comment (switch87) [inline snippets/cs.snippets]: Did the requested changes

**Comment by lpil** [comment]
Thanks. :)

**Pull request merged**